policies/strategic_planning_committee_draft.md

```diff
@@ -0,0 +1,56 @@
+Are you interested to join Astropy Strategic Planning Committee?
+Here is what to expect.
+
+## Time commitment
+
+* Meet at least monthly. Each meeting could be 30-60 minutes long.
+  Notes should be publicly available.
+* Must be willing to attend the annual Astropy Coordination Meeting, either
+  in-person or remotely. This is about a week. Or if you join remotely just
+  for certain sessions, then a few days.
+* Outside of meetings, must be willing to invest time in actually accomplishing
+  duties laid out below. Time estimate is tough here because it depends on
+  workload at a given time, and how that load is divided among committee members.
+  Maybe no more than a few days a month?
+* Some of the duties might involve even more meetings (e.g., reaching out to
+  specific communities). Time estimate is quite impossible to predict at this point.
+  No more than 1-2 weeks a year?
+
+## Expectations
+
+You have to be willing to perform duties as laid out in https://www.astropy.org/team#strategic_planner,
+i.e., do long-term planning for the Astropy Project as a whole, including:
+
+* Shape and provide a vision for next major steps in the Project, in consultation with the whole Astropy community.
+* Maintain an overview of all the project activities (both from developer and user perspectives).
+* Provide points of contact for external stakeholders on the above (with the Coordination Committee).
+* Monitor project activities to see how well aligned they are with the roadmap and strategic goals of the project, and report back to the community and Coordination Committee.
+* Coordinate writing grant proposals (in collaboration with the Finance Committee, Coordination Committee, and community).
+* Maintain the Astropy Project Roadmap.
+* Engage with, and collect input from, the whole Astropy Community on project vision and priorities.
+
+Deliverables based on Team page duties:
+
+* Progress reports, as needed. At the very least, annually during Astropy Coordination Meeting.
+* Grant proposals, as needed. At most once every few years?
+* Roadmap in https://github.com/astropy/astropy-project/tree/main/roadmap.
+* Project boards?
+* Documentation, communication, or assets necessary to accomplish duties above.
+
+Additional ongoing deliverables proposed by CoCo:
+
+* A plan for more support/involvement with radio? NRAO and SKA.
+* A plan for more administrative support, e.g, maintain email lists.
+* A plan for any additional initiatives the committee feels is important for Astropy's future.
+
+## Open questions/concerns
+
+* If no term limit and not elected position, will this committee be hoarded by a few people
+  and no opportunity for fresh prespective?

```

Comment:
CoCo liaison can keep an eye on its operations and determine any action needed should the need arises. If CoCo liaison itself is MIA, then other committee members can follow suit.